docs/tutorials/install_and_admin/geonode_install/install_geonode_application.txt

```diff
@@ -124,27 +130,20 @@ We are going to install GeoNode as a dependency of a **Customized DJango Project
 
 .. code-block:: bash
 
-    $ pip install Django==1.8.18
-    $ django-admin.py startproject --template=https://github.com/GeoNode/geonode-project/archive/2.8.0.zip -e py,rst,json,yml my_geonode
+    pip install Django==1.11.20
+    git clone https://github.com/GeoNode/geonode-project.git -b master
+    django-admin startproject --template=./geonode-project -e py,rst,json,yml,ini,env,sample -n Dockerfile my_geonode
 
 Let's install the GeoNode dependencies and packages into the Python Virtual Environment:
 
 .. code-block:: bash
 
-    $ cd my_geonode
-
-    # Find the closest pygdal version.
-    # Example: 2.2.1 ...  2.2.1.3, ...
-    $ gdal-config --version && pip install pygdal==
-
-    $ vim requirements.txt
-    # Make sure requirements contains reference to geonode 2.8 branch
-    # and correct gdal version (see above)
-    -e git://github.com/GeoNode/geonode.git@2.8.0#egg=geonode
-    pygdal==2.2.1.3
-
-    $ pip install -r requirements.txt --upgrade
-    $ pip install -e . --upgrade --no-cache
-
+    cd my_geonode
+    pip install -r requirements.txt --upgrade;
+    pip install -e . --upgrade --no-cache
+    # Install correct gdal version 
+    GDAL_VERSION=`gdal-config --version`
+    PYGDAL_VERSION="$(pip install pygdal==$GDAL_VERSION 2>&1 | grep -oP '(?<=: )(.*)(?=\))' | grep -oh $GDAL_VERSION\.[0-9])"
+    pip install pygdal==$PYGDAL_VERSION
```

Comment:
This section seems very similar to the documentation of the development installation. Is there a way to share common parts?

Reply:
I'm a bit afraid this could confuse users again( as the shared local_settings part currently does). **But sure if you have an idea? It's always welcome!**

What do you think to shorten this step a bit like

`$ gdal-config --version | cut -c 1-5 | xargs -I % pip install 'pygdal>=%.0,<=%.999'`

Better?